### PR TITLE
Add featureCategory field to package template

### PR DESCRIPTION
**Pull request by davidkline-ms**
This change adds a custom field to the package manifests. It adds the "featureCategory" field and sets the value to "Mixed Reality Toolkit". This tag will be used in the future and has no impact if a tool or process is not expecting to see it.

### CODE REVIEW

File: Assets/MRTK/Examples/packagetemplate.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
     "name": "com.microsoft.mixedreality.toolkit.examples",
     "displayName": "Mixed Reality Toolkit Examples",
+    "featureCategory": "Mixed Reality Toolkit",
```
Comment (keveleigh) [inline Assets/MRTK/Examples/packagetemplate.json:4]: Do we want to append this tag with `ms` or `msft` or something, or do you think the chances of this colliding with anything npm or Unity might add is pretty low?

Comment (davidkline-ms) [reply Assets/MRTK/Examples/packagetemplate.json:4]: that's not a bad thought! thx!

Comment (davidkline-ms) [reply Assets/MRTK/Examples/packagetemplate.json]: done

**Review by keveleigh — APPROVED** [review]

**Pull request merged**